Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Fix all currently missing Dynasty localization in dev

## Changelog:
- Fixes a number of Dynasties which had either missing or incorrect localization:
  - Satyr: Deth'ryll, Fallenroot, Nethervine, Shadowhoof
  - Azerothian: Fitzgerald
  - Gnomish: Arcfuzz, Arckettle, Arcspindle
  - Theramorian: Sagar, Pickering
  - Highborne: Twinshadow
  - Frostborn: Frost, Ice, Snow
  - Patron character: Mace'arosh


## Developer changelog:
- (All languages) Fix keys of 4/5 Xavic dynasty names that needed `dynn_` appended in wc_dynasty_names
- (English) Add localization for Fitzgerald/Arcfuzz/Arckettle/Arcspindle/Sagar/Frost/Ice/Snow dynasties
  - Add Arcfuzz/Arckettle/Arcspindle to 10000_gnome, commented out
  - Add Frost/Ice/Snow to 124000_frostborn, commented out
- (All languages) Fix misspelling of key "dyyn_twinshadow" to "dynn_twinshadow" for all languages, wc_dynasty_names
- (Eng/Fr/Ger) Fix localization key of patron dynasty from dynn_Mace_arosh to proper dynn_Mace'arosh, wc_dynasty_names
- Fix two missing quotation marks in wc_gilnean for name_list_theramorian, the cadet_dynasty_names and dynasty_names buckets. This was causing issues with localization for Sagar/Pickering.


## Tests:


# How to test:
Run game in observer mode and throughout periods of the game, check for any dynasties missing localization by going to the Character Search (C), setting Relation to All, and search for "dynn" or "dynn_".

To confirm fixes listed in changelog, you can also search for those dynasty names when they are spawned in the game.
